randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -177,17 +177,31 @@ Extra - asset_id: 4
   * Normal Door to The Moat/Door to West Ocean
   > Door to Wrecked Ship Entrance
       Any of the following:
-          # TODO: Check to see if possible with HBJ.
           Grapple Beam or Space Jump
-          # It is possible to jump across by not using full jumps to avoid bonking on the ceiling.
+          Horizontal Infinite Bomb Jump (Expert) and Can Lay Bombs
+          # It is possible to jump across by not using full jumps to avoid bonking on the celing.
           Movement (Intermediate)
           All of the following:
-              # From the sea floor
+              # From the Sea Floor
               Gravity Suit
               Any of the following:
                   Hi-Jump Boots or Wall Jumps (Beginner)
                   # IBJ
                   Vertical Infinite Bomb Jump (Beginner) and Can Lay Bombs
+  > Door from Gravity Suit Room
+      Any of the following:
+          Space Jump
+          All of the following:
+              # Get over and WJ up
+              Wall Jumps (Intermediate)
+              Grapple Beam or Movement (Intermediate)
+          All of the following:
+              # From the sea floor
+              Gravity Suit
+              Any of the following:
+                  Vertical Infinite Bomb Jump (Advanced) and Can Lay Bombs
```

Comment:
What about IBJing from either yellow or purple?
![grafik](https://github.com/randovania/randovania/assets/38186597/8b9f53e2-d2d0-4d98-87b3-96035441683c)

Reply:
Expert HBJ, i'll add it

Reply:
Added HBJ from purple path. 
Unsure if Yellow is possible, but if it is it would have the same requirements as from the purple path.